src/pages/workspace/distanceRates/PolicyDistanceRateTaxRateEditPage.tsx

```diff
@@ -0,0 +1,104 @@
+import type {StackScreenProps} from '@react-navigation/stack';
+import React, {useMemo} from 'react';
+import {Keyboard} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import type {OnyxEntry} from 'react-native-onyx';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import ScreenWrapper from '@components/ScreenWrapper';
+import SelectionList from '@components/SelectionList';
+import RadioListItem from '@components/SelectionList/RadioListItem';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import Navigation from '@libs/Navigation/Navigation';
+import type {SettingsNavigatorParamList} from '@navigation/types';
+import AccessOrNotFoundWrapper from '@pages/workspace/AccessOrNotFoundWrapper';
+import * as Policy from '@userActions/Policy/Policy';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+import type SCREENS from '@src/SCREENS';
+import type * as OnyxTypes from '@src/types/onyx';
+
+type ListItemType = {
+    value: string;
+    text: string;
+    isSelected: boolean;
+    keyForList: string;
+};
+
+type PolicyDistanceRateTaxRateEditPageOnyxProps = {
```

Comment:
We can use [WithPolicyOnyxProps](https://github.com/Expensify/App/blob/main/src/pages/workspace/withPolicy.tsx) here and in `PolicyDistanceRateTaxReclaimableOnEditPage.tsx`